README.md

```diff
@@ -1,8 +1,12 @@
 # FinOps Open Cost and Usage Specification (FOCUS) - Specification Working Group
 
-The Open Cloud Bill working group will develop a common, source-neutral schema of
-billing, cost, usage, and observability data mapped to a variety of cloud service provider and SaaS product
-sources, with metadata, dimensions, metrics, and measures for the source and common schema fields. As per the [Working Group Charter]() **Link TBC**
+## Overview
+
+The FinOps Open Cost and Usage Specification (FOCUS) is a community-driven effort to develop a standard schema for cloud and SaaS billing data. The primary goal of the FOCUS specification is to make it easier to understand, report on, and manage cloud costs. The FOCUS spec and schema is intended to be adaptable across a variety of cloud service provider and SaaS product sources, with metadata, dimensions, metrics, and example mappings between the source and common schema fields.
```

Comment:
```suggestion
The FinOps Open Cost and Usage Specification (FOCUS) is a community-driven effort to develop a standard schema for billing data (Cloud, SaaS, and other billing datasets). The FOCUS specification is designed to assist FinOps practitioners in performing common FinOps capabilities by utilizing a standardized set of instructions, regardless of the billing data generator. The FOCUS specification defines columns (dimensions, metrics), column-specific requirements, and attributes (spec-wide requirements). This repository also provides supporting content that includes example mappings between well-known provider datasets and what's defined in the data.
```

Reply:
I'm going to add another commit mostly to kick the CLA bot since I added an add'l email address to my Github account.